stdlib/glob.pyi

```diff
@@ -2,14 +2,22 @@ import sys
 from _typeshed import StrOrBytesPath
 from collections.abc import Iterator, Sequence
 from typing import AnyStr
+from typing_extensions import deprecated
 
 __all__ = ["escape", "glob", "iglob"]
 
 if sys.version_info >= (3, 13):
     __all__ += ["translate"]
 
-def glob0(dirname: AnyStr, pattern: AnyStr) -> list[AnyStr]: ...
-def glob1(dirname: AnyStr, pattern: AnyStr) -> list[AnyStr]: ...
+if sys.version_info >= (3, 13):
+    @deprecated("Will be removed in Python 3.15; Use `glob.glob` and pass *root_dir* argument instead.")
+    def glob0(dirname: AnyStr, pattern: AnyStr) -> list[AnyStr]: ...
+    @deprecated("Will be removed in Python 3.15; Use `glob.glob` and pass *root_dir* argument instead.")
```

Comment:
in general these functions have already been removed from the main branch, so maybe it's worth writing `"Removed in ..."`
```suggestion
    @deprecated("Removed in Python 3.15; Use `glob.glob` and pass *root_dir* argument instead.")
    def glob0(dirname: AnyStr, pattern: AnyStr) -> list[AnyStr]: ...
    @deprecated("Removed in Python 3.15; Use `glob.glob` and pass *root_dir* argument instead.")
```